maingame.py

```diff
@@ -0,0 +1,688 @@
+import os, sys
```

Comment:
Header comment with your name and short description of this module would be great.